angular.json

```diff
@@ -17,18 +21,29 @@
         "build": {
           "builder": "@angular-devkit/build-angular:browser",
           "options": {
+<<<<<<< HEAD
             "outputPath": "dist/landville-frontend",
+=======
+            "outputPath": "dist/landvile-frontend",
+>>>>>>> 167132923-ch(project-boilerplate): Setup boilerplate for the frontend
```

Comment:
@kelvinndmo, You pushed changes halfway through the rebase process, could you please rectify this.

Reply:
damn sure.On it